src/dwarf/Gfind_proc_info-lsb.c

```diff
@@ -135,9 +135,9 @@ load_debug_frame (const char *file, char **buf, size_t *bufsize, int is_local,
 #if defined(SHF_COMPRESSED)
   if (shdr->sh_flags & SHF_COMPRESSED)
     {
-      unsigned long destSize;
       Elf_W (Chdr) *chdr = (shdr->sh_offset + ei.image);
 #ifdef HAVE_ZLIB
+      unsigned long destSize;
```

Comment:
Unrelated change: gcc was warning that destSize is unused (it is unused outside of HAVE_ZLIB branch).